muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/chat/ChatRESTService.java

```diff
@@ -287,4 +298,45 @@ public Response createOrUpdateUserChatSettings(UserChatSettings userChatSettings
 	}
 	return Response.ok(userChatSettings).build(); 
   }
+  
+  @GET
+  @Path("/workspaceChatSettings/{WorkspaceEntityId}")
+  @RESTPermit(handling = Handling.INLINE)
+  public Response workspaceChatSettingsGet(@PathParam("WorkspaceEntityId") Long workspaceEntityId, WorkspaceChatSettings workspaceChatSettings) {
```

Comment:
* WorkspaceChatSettings parameter is not used in this method.
* Also REST GET with id usually returns the resource (WorkspaceChatSettings here) (i.e. use the workspaceEntityId as parameter to fetch the WorkspaceChatSettings from a Controller and return it with Response.ok...).